src/renderer/ipc.ts

```diff
@@ -3,11 +3,30 @@
  * to be able to switch this layer later on...
  */
 
-import { ipcRenderer } from 'electron'
-const log = require('../shared/logger').getLogger('renderer/ipc')
+import { ipcRenderer, remote, clipboard as _clipboard } from 'electron'
+import { ExtendedApp, AppState } from '../shared/shared-types'
+import logger from '../shared/logger'
+
+const log = logger.getLogger('renderer/ipc')    
 
 export const ipcBackend = ipcRenderer
 
+export const appState: AppState = (remote.app as ExtendedApp).state 
+
+export const showOpenDialog = remote.dialog.showOpenDialog  
+
+export const openExternal = remote.shell.openExternal
+
+export const openItem = remote.shell.openItem
+
+export const showSaveDialog = remote.dialog.showSaveDialog
+
+export const showMessageBox = remote.dialog.showMessageBox
+
+export const getPath = remote.app.getPath 
+
+export const clipboard = _clipboard
```

Comment:
use navigator.clipboard instead, we don't need the electron variant, anyway would be better If you had opened this on top of my preload pr.....